crates/common/mqtt_channel/src/config.rs

```diff
@@ -121,12 +121,12 @@ impl Debug for ClientAuthConfig {
     }
 }
 
-#[derive(Debug, Clone)]
-struct PrivateKey(rustls::PrivateKey);
+#[derive(Debug)]
+struct PrivateKey(rustls::pki_types::PrivateKeyDer<'static>);
 
 impl zeroize::Zeroize for PrivateKey {
     fn zeroize(&mut self) {
-        self.0 .0.zeroize()
+        // FIXME I can't be implemented with rustls 0.23 due to immutability
```

Comment:
One can submit a PR on `rustls_pki_types`

Reply:
Since this is related to security and a removal of functionality, we should create a ticket to track it (e.g. and create a PR as @didier-wenzek suggested) before merging so we don't forget about this.